README.markdown

```diff
@@ -22,3 +22,51 @@ long the outcome is distributed under the same license. Otherwise, you must
 obtain a [commercial license](./LICENSE-COMMERCIAL) that removes such
 restrictions. Read more about our licensing approach
 [here](https://www.sourcemeta.com/licensing/).
+
+Example
+--------------
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()
+{
+    // creating json using parse
+    sourcemeta::jsontoolkit::JSON _template_ =
+        sourcemeta::jsontoolkit::parse(R"JSON({
+            "name" : "xyz" ,
+            "age" : 20 ,
+            "address" : "zxy"
+        })JSON");
+
+    // stringstream for output
+    std::ostringstream stringvar;
+    // pointer for the name
+    const sourcemeta::jsontoolkit::Pointer name_pointer{"name"};
+    const sourcemeta::jsontoolkit::Pointer age_pointer{"age"};
+    const sourcemeta::jsontoolkit::Pointer address_pointer{"address"};
+    // value instance for json
```

Comment:
This comment makes little sense too as every instance is a JSON value by definition. Maybe what you wanted to say was "A new JSON document for the `name` property"